unlock-app/src/components/interface/Error.js

```diff
@@ -2,49 +2,53 @@ import { connect } from 'react-redux'
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { setError } from '../../actions/error'
+import { resetError } from '../../actions/error'
 import Buttons from './buttons/layout'
 
-export const Error = ({ children, error, close }) => {
-  const content = children || error
-  if (!content) {
+export const Errors = ({ children, errors, close }) => {
+  const content =
+    children ||
+    (errors && errors.map(error => <Error key={error}>{error}</Error>))
+  if (!content || !content.length) {
     return null
   }
   return (
     <Wrapper>
       {content}
-      <Buttons.Close as="button" onClick={close} size="16px">
-        X
-      </Buttons.Close>
+      <SecondColumn cols={errors.length ? errors.length : 1}>
```

Comment:
yes, but this is a question I had. The original PR, as you noted, had separate close buttons for each error, as well as something for the whole. I implemented it this way because of a combination of comments that you and Sascha had made. I have no opinion either way. If you want it to be close-each-error-individually-and-possibly-all, I'll do that, and pass on the requirements to Sascha to make a pretty design for it. So let's discuss that whenever you decide what is best

Reply:
yes, this could change pending discussion of what we want, I have no opinion